packages/chat-client/src/controllers/engine.ts

```diff
@@ -371,6 +374,39 @@ export class ChatEngine extends IChatEngine {
     return inviteId;
   };
 
+  public unregisterIdentity: IChatEngine["unregisterIdentity"] = async ({
+    account,
+  }) => {
+    const iat = Date.now();
+    const keys = this.client.chatKeys.get(account);
+    const didPublicKey = composeDidPkh(account);
+    const unregisterIdentityPayload: InviteKeyClaims = {
+      iat,
+      exp: jwtExp(iat),
```

Comment:
Is this how unregistering works ? By issuing a token that expires at the current time ?